ScamSearcher/test_settings.py

```diff
@@ -1,5 +1,5 @@
 """This is additional settings config for testing only."""
-from ScamSearcher.settings import *
+from ScamSearcher.settings import *  # pylint: disable=w0401,w0614
```

Comment:
__[flake8]__
* [F403] 'from ScamSearcher.settings import *' used; unable to detect undefined names